data.json

```
@@ -233,6 +233,15 @@
     "dateAdded": "2018-02-19T09:11:04.000Z",
     "tags": ["accessories", "stickers"]
   },
+  {
+    "name": "DocsGPT",
+    "difficulty": "medium",
+    "description": "Contribute to code and make a valid PR or improve the documentation by fixing the existing documentation or by creating a guide or improve the UI/UX or design a new feature.",
+    "reference": "https://github.com/arc53/DocsGPT/blob/main/HACKTOBERFEST.md",
+    "image": "https://res.cloudinary.com/daily-now/image/upload/f_auto,q_auto/v1/posts/3149030bec9354a75cbfefedfdb5b650",
```

```suggestion
    "image": "blob:https://imgur.com/10003162-e67f-4cbf-8929-c902416cfc55",
```
Here's an alternative imag